tests/chainer_tests/test_link_hook.py

```diff
@@ -179,6 +179,12 @@ def test_local_hook_named(self):
     def test_local_hook_unnamed(self):
         self._check_local_hook(None, 'MyLinkHook')
 
+    def test_link_addhook(self):
```

Comment:
Let's give a more explicit name.
```suggestion
    def test_addhook_returns_self(self):
```